rpmlint/checks/MenuXDGCheck.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, config, output):
         # desktop file need to be in $XDG_DATA_DIRS
         # $ echo $XDG_DATA_DIRS/applications
         # /var/lib/menu-xdg:/usr/share
-        super().__init__(config, output, r'(?:/usr|/etc/opt|/opt/.*)/share/applications/.*\.desktop$')
+        super().__init__(config, output, r'(?:/usr/share)/applications/.*\.desktop$')
```

Comment:
Why instead of '(?:/usr/share)/applications/' ust not use '/usr/share/applications/'

Reply:
because I am confused :)